test/integration/SchemeRegistrar.spec.ts

```diff
@@ -171,7 +171,7 @@ describe('SchemeRegistrar', () => {
 
         const executedIsIndexed = async () => {
           return (await sendQuery(getSchemeRegistrarProposalExecuteds)).schemeRegistrarProposalExecuteds.length
-           > prevExecutedsLength;
+           > prevExecutedsLength + 1;
```

Comment:
why and how this pr effect getSchemeRegistrarProposalExecuteds length ?

Reply:
That's a bug I fixed is not exactly related.